src/frontends/pytorch/src/op/batch_norm.cpp

```diff
@@ -15,12 +15,36 @@ OutputVector translate_batch_norm(NodeContext& context) {
     // Schema: aten::batch_norm(Tensor input, Tensor? weight, Tensor? bias, Tensor? running_mean, Tensor? running_var,
     // bool training, float momentum, float eps, bool cudnn_enabled) -> Tensor
     auto input = context.get_input(0);
-    auto weight = context.get_input(1);
-    auto bias = context.get_input(2);
-    auto running_mean = context.get_input(3);
-    auto running_var = context.get_input(4);
+    auto input_shape = context.mark_node(std::make_shared<opset8::ShapeOf>(input));
+    Output<Node> weight;
+    Output<Node> bias;
+    if (!context.input_is_none(1)){ 
+        weight = context.get_input(1);
+    }
+    else {
+        auto zero_i = context.mark_node(opset8::Constant::create(element::i64, Shape{}, {0}));
```

Comment:
Can do this broadcasting as function, to reduce code duplication.

Reply:
added